client/src/Components/VerticalNavBarComponents/Sorting.js

```diff
@@ -0,0 +1,55 @@
+import { useCart } from "../../cart-context/cart-context";
+
+export const Sorting = () => {
+  const { cartState, cartDispatch } = useCart();
+  return (
+    <>
+      <div
+        className='opt-clear-all'
+        onClick={() => cartDispatch({ type: "CLEARALLFILTERS" })}>
```

Comment:
can add separators in dispatch type to improve readability
instead of : CLEARALLFILTERS,
you can use : CLEAR_ALL_FILTERS

Reply:
yeah I have resolved it now